src/Cli/dotnet/commands/dotnet-tool/run/LocalizableStrings.resx

```diff
@@ -129,6 +129,9 @@
   <data name="CommandNameArgumentName" xml:space="preserve">
     <value>COMMAND_NAME</value>
   </data>
+  <data name="RollForwardDescription" xml:space="preserve">
+    <value>Roll forward to framework version  (LatestPatch, Minor, LatestMinor, Major, LatestMajor, Disable).</value>
+  </data>
```

Comment:
I think we can delete this string (which doesn't take into account the change to `--allow-roll-forward`) and use the same `RollForwardOptionDescription` string from the install command.  You should be able to qualify it with the right namespace when you use it so that you don't have to duplicate the string in the resource files.